bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestCreateTable.java

```diff
@@ -176,5 +175,132 @@ public void testSplitKeys() throws IOException {
         LOG.warn("Error cleaning up the table", t);
       }
     }
+  }
+
+  @Test
+  public void testEvenSplitKeysFailures() throws IOException {
+    Admin admin = getConnection().getAdmin();
+    TableName tableName = TableName.valueOf("TestTableWithSplits");
+    HTableDescriptor descriptor = new HTableDescriptor(tableName);
+    descriptor.addFamily(new HColumnDescriptor(COLUMN_FAMILY));
+    byte[] startKey = Bytes.toBytes("AAA");
+    byte[] endKey = Bytes.toBytes("ZZZ");
+
+    try {
+      admin.createTable(descriptor, startKey, endKey, 2);
+      Assert.fail();
+    }
+    catch (IllegalArgumentException e) {
+    }
+    try {
+      admin.createTable(descriptor, endKey, startKey, 5);
+      Assert.fail();
+    }
+    catch (IllegalArgumentException e) {
+    }
+  }
+
+  @Test
+  public void testThreeRegionSplit() throws IOException {
+
+    Admin admin = getConnection().getAdmin();
+
+    TableName tableName = TableName.valueOf("TestTableWithThreeSplits");
```

Comment:
Add  UUID.randomUUID().toString() to the end of the table name.  We sometimes run concurrent integration-test jobs against the same cluster.